src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/Reference.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <Wizard
+    :ref="'wizard'"
+    :wizard="wizardStore.wizardData?.inviteType === PortalInviteType.WORK_EXPERIENCE ? workExperienceReferenceWizardConfig : characterReferenceWizardConfig"
```

Comment:
I tried to link this directly to the wizard, however vue would throw this warning message for a reactive component. 

![image](https://github.com/bcgov/ECC-ECER/assets/74216496/6998697b-646c-4819-b912-afd74a19b5a3)

Seems to go away if we input the config based with a conditional.